src/implementations/minecraft/configurable.rs

```diff
@@ -20,6 +24,14 @@ impl TConfigurable for Instance {
         self.config.name.clone()
     }
 
+    fn flavour(&self) -> MaybeUnsupported<String> {
```

Comment:
this dont feel intentional, Minecraft Server should have flavours right?

Reply:
timeout_last_left is a muted smart pointer and must be explicitly dereferenced

Reply:
Fixed in [08f86bc](https://github.com/Lodestone-Team/client/pull/42/commits/08f86bcac73c003a21ebb45ef5d61f224ddfb6f5)